test/e2e/framework.go

```diff
@@ -30,6 +30,7 @@ import (
 	"time"
 
 	"github.com/containernetworking/plugins/pkg/ip"
+	log "github.com/sirupsen/logrus"
```

Comment:
we should unify logging for the e2e tests, but we don't typically use logrus here